packages/reports/src/Components/Form/CompositeCalendar/composite-calendar-mobile.tsx

```diff
@@ -34,26 +43,44 @@ export const RadioButton = ({ id, className, selected_value, value, label, onCha
 };
 const CUSTOM_KEY = 'custom';
 
+type TCompositeCalendarMobile = {
+    input_date_range: TInputDateRange;
+    current_focus: string;
+    duration_list: Array<TInputDateRange>;
+    onChange: (
+        value: { from: moment.Moment | null; to: moment.Moment; is_batch: boolean },
+        extra_data: { date_range: any }
```

Comment:
```suggestion
        extra_data: { date_range: TInputDateRange }
```
not sure :thinking:

Reply:
Can be done cause it has the necessary properties